org.eclipse.xtext.xbase.lib/src/org/eclipse/xtext/xbase/lib/IteratorExtensions.java

```diff
@@ -1043,4 +1044,25 @@ protected T computeNext() {
 		}
 		return max;
 	}
+
+    /**
```

Comment:
Please add a not that this will advance the iterator or may even exhaust it.

Reply:
I added the note as well to
- exists
- elementsEqual
- min, minBy
- max, maxBy
- contains

Reply:
Appreciated!